examples/java-agent/pom.xml

```diff
@@ -77,7 +77,9 @@
           <to>
             <image>gcr.io/REPLACE-WITH-YOUR-GCP-PROJECT/image-built-with-jib</image>
           </to>
-          <extraDirectory>${agent-extraction-root}</extraDirectory>
+          <extraDirectories>
+            <paths>${agent-extraction-root}</paths>
+          </extraDirectories>
           <container>
             <ports>4567</ports>
```

Comment:
does this need to be modified for cloud run $PORT?

Reply:
We can't actually change this on the fly though.  The default for SparkJava is to listen on port 4567.  Cloud Run passes in a PORT and the container is required to listen on that port; Cloud Run doesn't care about the container image's exposed ports.

Let me add a note to the source and here to this effect.  I also forgot to update the build.gradle to use the `extraDirectories`.